ts/boot/scheduleLocalNotifications.ts

```diff
@@ -0,0 +1,57 @@
+import { addDays, addMonths, addWeeks } from "date-fns";
+import I18n from "i18n-js";
+import PushNotification from "react-native-push-notification";
+import { store } from "../App";
+import { updateLocalNotificationsScheduled } from "../store/actions/notifications";
+import { notificationsLocalScheduledSelector } from "../store/reducers/notifications/localScheduled";
+
+const FIRST_ACCESS_SPID_TAG: string = "local_notification_spid";
+
+/*
+ * Schedule a set of local notifications to remind the user to authenticate with spid
+ */
+export const scheduleLocalNotificationsAccessSpid = () => {
+  const isToSchedule = notificationsLocalScheduledSelector(store.getState())
+    .isToSchedule;
+  if (isToSchedule) {
+    const nowDate = new Date();
+    // Configure all the dates to schedule local notifications
+    const oneDayDate = addDays(nowDate, 1);
+    const threeDayDate = addDays(oneDayDate, 3);
```

Comment:
shouldn't this be oneDayDate + 1 day = threeDayDate

also the following schedules seem wrong

Reply:
In the commit [c72c98b](https://github.com/teamdigitale/io-app/pull/992/commits/c72c98ba0fc7eb05c8121a303b5fd865983ca119) I changed the name of the variables to make it more understandable.
In the doc [CDM-10](https://docs.google.com/document/d/1iKsNAqNcDzImQjEDD8tJMHgzyPOFHmh9IUTq8pNpXW8/edit) as a prerequisite, the scheduling dates always referred to the date of the previous notification.
If your observation was linked to something else, can you give me more details?